package.json

```diff
@@ -38,39 +40,45 @@
     "@types/rsvp": "^4.0.2",
     "@typescript-eslint/eslint-plugin": "^1.13.0",
     "@typescript-eslint/parser": "^1.13.0",
+    "babel-eslint": "^10.0.3",
+    "chai": "^4.2.0",
     "documentation": "^9.3.1",
-    "ember-cli": "~3.8.1",
+    "ember-cli": "~3.14.0",
+    "ember-cli-babel-plugin-helpers": "^1.1.0",
     "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-htmlbars": "^3.0.0",
-    "ember-cli-htmlbars-inline-precompile": "^2.1.0",
-    "ember-cli-inject-live-reload": "^1.8.2",
+    "ember-cli-htmlbars": "^4.0.5",
+    "ember-cli-inject-live-reload": "^2.0.1",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-test-loader": "^2.2.0",
     "ember-cli-typescript": "^1.5.0",
-    "ember-cli-uglify": "^2.1.0",
+    "ember-cli-uglify": "^3.0.0",
+    "ember-concurrency": "^1.1.2",
+    "ember-concurrency-decorators": "^1.0.0",
     "ember-disable-prototype-extensions": "^1.1.3",
     "ember-export-application-global": "^2.0.0",
-    "ember-load-initializers": "^1.1.0",
+    "ember-load-initializers": "^2.1.0",
     "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-resolver": "^5.0.1",
-    "ember-source": "~3.7.0",
-    "ember-source-channel-url": "^1.1.0",
+    "ember-resolver": "^5.3.0",
+    "ember-source": "~3.14.1",
+    "ember-source-channel-url": "^2.0.1",
     "ember-try": "^1.2.1",
     "eslint": "^5.14.1",
     "eslint-config-prettier": "^6.0.0",
-    "eslint-plugin-ember": "^5.2.0",
-    "eslint-plugin-node": "^8.0.1",
+    "eslint-plugin-ember": "^7.1.0",
+    "eslint-plugin-node": "^10.0.0",
     "eslint-plugin-prettier": "^3.0.1",
     "loader.js": "^4.7.0",
+    "mocha": "^6.2.2",
     "prettier": "^1.18.2",
     "qunit": "^2.9.2",
+    "tmp": "^0.1.0",
     "typescript": "^3.5.3"
   },
   "resolutions": {
     "**/engine.io": "~3.3.0"
   },
   "engines": {
-    "node": "6.* || 8.* || >= 10.*"
+    "node": "8.* || >= 10.*"
```

Comment:
I'd prefer that we did this change separately, in a different PR. Bumping the engine version, which I know is necessary here, requires a major version bump. Node 8 is almost out of LTS, so we'll have to do another one then also. Perhaps we can figure out how to avoid 2 majors in 1 month.

Reply:
FYI, many of the changes that you've requested go in a separate PR are all the result of `ember-cli-update` applying the latest addon blueprint, so they can all collapse into https://github.com/rwjblue/ember-test-waiters/pull/81#discussion_r353887193

Reply:
Yep, it'd be awesome to do those maintenance type tasks separately, as they're adding a bit of extra clutter to this PR. Does this seem fair?